bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/model/InverterData.java

```diff
@@ -82,6 +84,37 @@ default short getBatteryTemperature() {
         return Short.MIN_VALUE;
     }
 
+    default short getInverterTemperature1() {
+        return Short.MIN_VALUE;
+    }
+
+    default short getInverterTemperature2() {
+        return Short.MIN_VALUE;
+    }
+
+    default short getInverterWorkModeCode() {
+        return Short.MIN_VALUE;
+    }
+
+    @Nullable
+    default String getInverterWorkMode() {
+        switch (getInverterWorkModeCode()) {
+            case 0:
+                return WORKMODE_WAITING;
+            case 1:
+                return WORKMODE_CHECKING;
+            case 2:
+                return WORKMODE_NORMAL;
+            case 3:
+                return WORKMODE_FAULT;
+            case 4:
+                return WORKMODE_PERMANENT_FAULT;
+            case 5:
+                return WORKMODE_UPGRADING;
+        }
```

Comment:
Add a default state instead of returning a null value.
```suggestion
       default: return WORKMODE_UNKNOWN;
        }
```